packages/orchestration/src/facade.js

```diff
@@ -94,6 +99,8 @@ const makeLocalChainFacade = localchain => {
 };
 
 /**
+ * XXX should be provide...Facade
```

Comment:
alternately keep this but also export `provideOrchestrationFacade` that ensures it runs at most once per zone.

Reply:
`provide` isn't straightforward: `maker return value {"orchestrate":"[Function orchestrate]"} is not storable`

Reply:
the stored parts would just be the kinds. The facade need not be memoized since it's stateless.